src/IssueViz.Security/HotspotsList/HotspotsControl.xaml

```diff
@@ -4,20 +4,148 @@
              xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
              xmlns:wpf="clr-namespace:SonarLint.VisualStudio.Core.WPF;assembly=SonarLint.VisualStudio.Core"
              xmlns:issueVisualizationControl="clr-namespace:SonarLint.VisualStudio.IssueVisualization.IssueVisualizationControl;assembly=SonarLint.VisualStudio.IssueVisualization"
+             xmlns:vsShell="clr-namespace:Microsoft.VisualStudio.Shell;assembly=Microsoft.VisualStudio.Shell.14.0"
+             xmlns:vsCatalog="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.ImageCatalog"
+             xmlns:vsImaging="clr-namespace:Microsoft.VisualStudio.Imaging;assembly=Microsoft.VisualStudio.Imaging"
+             xmlns:vsTheming="clr-namespace:Microsoft.VisualStudio.PlatformUI;assembly=Microsoft.VisualStudio.Shell.14.0"
              DataContext="{Binding ViewModel, RelativeSource={RelativeSource Mode=Self}}">
     <UserControl.Resources>
+        <vsTheming:BrushToColorConverter x:Key="BrushToColorConverter"/>
+        <issueVisualizationControl:SpanToNavigabilityConverter x:Key="SpanToNavigabilityConverter"/>
         <wpf:AggregatorValueConverter x:Key="SpanToInvertedVisibilityConverter">
             <issueVisualizationControl:SpanToNavigabilityConverter />
             <wpf:BoolToVisibilityConverter TrueValue="Collapsed" FalseValue="Visible" />
         </wpf:AggregatorValueConverter>
 
+        <Style x:Key="ColumnHeaderGripperStyle" TargetType="{x:Type Thumb}">
```

Comment:
Is this based on an existing/example implementation?

Reply:
Yep, PART_RightHeaderGripper and PART_LeftHeaderGripper are part of the default DataGridColumnHeader template.